app/authorization/client/hasPermission.js

```diff
@@ -3,7 +3,9 @@ import { Template } from 'meteor/templating';
 import * as Models from '../../models';
 import { ChatPermissions } from './lib/ChatPermissions';
 
-function atLeastOne(permissions = [], scope) {
+function atLeastOne(permissions = [], scope, userId = undefined) {
```

Comment:
I think we don't need this default parameter as `undefined`, because if the user does not provide the value, anyway it will be undefined. :thinking: